luzer/luzer.c

```diff
@@ -324,55 +328,177 @@ load_custom_mutator_lib(void) {
 	return 0;
 }
 
+/* Structure for convenient argument parsing. */
+struct args {
+	char **argv;
+	int argc;
+};
```

Comment:
I would put all functions related to arg to separated files, for example `luzer_args.{c,h}`.

Reply:
good idea!

Reply:
created 2 files:
* luzer_args.c
* luzer_args.h